app/backend/src/clj/teet/land/owner_opinion_export.clj

```diff
@@ -0,0 +1,45 @@
+(ns teet.land.owner-opinion-export
+  (:require [teet.util.collection :as cu]
+            [hiccup.core :as h]
+            [teet.localization :refer [tr tr-enum with-language]]
+            [datomic.client.api :as d]
+            [teet.db-api.db-api-large-text :as db-api-large-text]))
+
+(defn- opinions-by-type
+  [db activity type]
+  (->>
+    (d/q '[:find (pull ?opinion [*])
+           :where
+           [?opinion :land-owner-opinion/activity ?activity]
+           [?opinion :land-owner-opinion/type ?type]
+           [(missing? $ ?opinion :meta/deleted?)]
+           :in $ ?activity ?type]
+         db activity type)
+    (db-api-large-text/with-large-text #{:cooperation.response/content :cooperation.opinion/comment})
+    (map first)))
+
+(defn summary-table [db activity type]
+  (let [opinions (opinions-by-type db activity type)]
+    (h/html
+      (with-language :et
+                     (cu/eager
+                       [:html
+                        [:head
+                         [:title (tr-enum type)]
+                         [:script
+                          (str "function copyToClipboard() {"
```

Comment:
copied from cooperation export? should have some "html export utils" ns?

Reply:
yea I was thinking the same, but decided not to do it yet, but maybe since it's already the second place this is needed in maybe it should be created now.

Reply:
Third time's the charm?